crates/vm/levm/src/opcode_handlers/stack_memory_storage_flow.rs

```diff
@@ -130,15 +129,25 @@ impl VM {
         key.to_big_endian(&mut bytes);
         let key = H256::from(bytes);
 
+        let mut base_dynamic_gas: U256 = U256::zero();
+
         let current_value = if self.cache.is_slot_cached(&address, key) {
+            // If slot is warm (cached) add 100 to base_dynamic_gas
+            base_dynamic_gas += U256::from(100);
+
             self.cache
                 .get_storage_slot(address, key)
                 .unwrap_or_default()
```

Comment:
Maybe we can clarify that this shouldn't fail because we already know it is cached from the if with an `expect` instead of using `unwrap_or_default`